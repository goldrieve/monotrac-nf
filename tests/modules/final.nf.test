nextflow_process {

    name "Test Process FINAL"
    script "main.nf"
    process "FINAL"

    test("Should combine predictions") {

        when {
            process {
                """
                input[0] =[
                    file("${projectDir}/tests/data/108AT_test_prediction.txt"),
                    file("${projectDir}/tests/data/SVP_test_prediction.txt")]
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }
    }
}