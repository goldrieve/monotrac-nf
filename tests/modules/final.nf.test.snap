{
    "Should combine predictions": {
        "content": [
            {
                "0": [
                    "combined_prediction.csv:md5,188ba0906d35114030f45662a7093568"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.7"
        },
        "timestamp": "2025-10-06T14:31:02.752931409"
    }
}